README.md

```diff
@@ -166,8 +166,29 @@ Your resulting code should look as follows:
 5. Click on the **Call Web API** and see the textual representation of the JSON object that is returned. Make sure your Node.js Web API sample is still running on port 5000. 
 6. Sign out by clicking the **Logout** button.  
 
+## Advance topics

```

Comment:
@sameerag  / @pkanher617  review for accuracy

Reply:
@negoe @jmprieur Should we also add sid/account params which we support as am alternative to login_hint here or is it too much information?

Reply:
@sameerag : if sid works better, then yes

Reply:
It's mostly the new API: We have sid || login_hint || account (object) and we recommend only one option passed. We generate the hints based on this info and there is a priority on which hints we pick and discard depending on the data available.

I will work with @negoe to document this clearly.